hackattoni.json

```diff
@@ -0,0 +1,5 @@
+{
+	"fork": "",
```

Comment:
Maybe we should put examples (or some defaults here for fork and remote) instead of empty strings.